docs/release-notes/release-notes-0.5.0.md

```diff
@@ -0,0 +1,234 @@
+
+- Swap options may be set at time of placement of each order rather than in the wallet settings.  This includes pre-sizing funds to avoid overlocking of UTXOs, and specifying higher network fee rates. (<https://github.com/decred/dcrdex/pull/1170>, <https://github.com/decred/dcrdex/pull/1669>)
+
+- Improved send/withdraw controls, allowing a complete sweep of the wallet minus transaction fees, or sending a precise amount like most wallets. (<https://github.com/decred/dcrdex/pull/1611>)
+
+- Bitcoin swap acceleration using CPFP transactions to increase the swap fee rate in the event of a network fee spike. This has not been in demand much recently, but we have encountered rapid increases in Bitcoin fee rates in the past where this is valuable. (<https://github.com/decred/dcrdex/pull/1555>)
+
+- Fiat currency value display. When enabled by the user, current USD exchange rates for all assets will be used to show equivalent fiat values on the UI. (<https://github.com/decred/dcrdex/pull/1600>)
+
+- QR code display for wallet receiving addresses. (<https://github.com/decred/dcrdex/pull/1483>)
+
+- There is now a "copy" button next to the wallet's receiving address. (<https://github.com/decred/dcrdex/pull/1639>, <https://github.com/decred/dcrdex/pull/1675>)
+
+- Rescan and recover functionality for the native BTC SPV wallet. You may also now reconfigure the native BTC wallet’s “birthday” to change the scope of the rescan. (<https://github.com/decred/dcrdex/pull/1507>, <https://github.com/decred/dcrdex/pull/1439>)
+
+- Add the option to obtain Decred network fee rates from dcrdata when the client wallet is unable. (<https://github.com/decred/dcrdex/pull/1654>)
+
+- Bitcoin “descriptor” wallets are now supported. This only applies to users of the Bitcoin Core software, not the native BTC wallet, who also choose to create this type of wallet that is becoming the default in future versions of Bitcoin Core. Support for Bitcoin Core v23, dropped support for v0.19. (<https://github.com/decred/dcrdex/pull/1659>, <https://github.com/decred/dcrdex/pull/1735>)
+
+- A DEX server's host name and TLS certificates may now be updated. (<https://github.com/decred/dcrdex/pull/1605>, <https://github.com/decred/dcrdex/pull/1602>)
+
+- When a wallet regains peers after having lost connectivity, a notification is displayed. (<https://github.com/decred/dcrdex/pull/1701>, <https://github.com/decred/dcrdex/pull/1719>)
+
+- Reconfiguring BTC wallets is now less disruptive as it only requires an internal restart of the wallet and chain service when absolutely required. (<https://github.com/decred/dcrdex/pull/1686>)
+
+- The user interface files are now embedded in the "dexc" executable.  To use the DEX client, only the one file is required. Any "site" folders left from previous versions may be deleted. (<https://github.com/decred/dcrdex/pull/1710>)
```

Comment:
Should call this out in "Upgrading" so they know nothing is missing, and that they can delete the old site folder.